arm/vrb_test.c

```diff
@@ -0,0 +1,76 @@
+#include <inttypes.h>
+typedef uint32_t u32;
+typedef uint64_t u64;
+typedef int32_t i32;
+typedef int64_t i64;
```

Comment:
I don't think these are worth defining.

Reply:
Those may or may not have been copy pasted in from one of my C test templates lol